docs/target-compatibility.md

```diff
@@ -161,3 +162,10 @@ On GLSL and VK targets loop unrolling uses the [GL_EXT_control_flow_attributes](
 
 Slang does have a cross target mechanism to [unroll loops](language-reference/06-statements.md), in the section `Compile-Time For Statement`.
 
+## Atomics on RWBuffer
```

Comment:
Are we sure that atomics on `RWBuffer` (as opposed to `RWStructuredBuffer` or `RWByteAddressBuffer`) are legal at all?

The `RWBuffer` type is a "formatted" resource, like a texture, and as such I wouldn't expect it to support atomic operations even when it is mutable (because the atomics would potentially have to perform format conversion).